run.sh

```diff
@@ -69,11 +65,17 @@ else
     echo "MongoDB installed"
 fi
 
-if [ -d "$PERSITENT_DIR/db" ]; then
-    echo "Persistence store exists at $PERSITENT_DIR/db"
+if [ -f "$PERSITENT_DIR/db/mongod.lock" ]; then
+    echo "Database exists at $PERSITENT_DIR/db. Not bootstrapping users."
+    BOOTSTRAP_USERS=0
 else
-    echo "Creating persistence store exists at $PERSITENT_DIR/db"
+    echo "Creating database location at $PERSITENT_DIR/db"
     mkdir -p $PERSITENT_DIR/db
+    if ! [ -f "bootstrap.json" ]; then
+        echo "Cannot bootstrap users. Please create bootstrap.json from bootstrap.json.sample."
+        exit 1
+    fi
+    BOOTSTRAP_USERS=1
```

Comment:
Agreed. I realized the tradeoff when I made the change, but I didn't want to require a file that may not even get used. Now, I realize that I can just move the entire db check to the top and we're fine.